src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -596,6 +631,7 @@ def load_status_file_components(self, initial_load=False):
                     self.__installation_substatus_json = complete_status_file_data['status']['substatus'][i]
                 else:
                     self.__installation_summary_json = self.__get_substatus_message(complete_status_file_data, i)
+                    # Reload patches into installation ordered map for fast look up
```

Comment:
Marking all changes not related to this PR goal. See my other comment on what can be done with these

Reply:
its' counterproductive to revert this change, this logic is very small and straightforward, so I refactor the logic into a function and reuse it in truncation to avoid replication.

Reply:
Ignore this for now to look at the bigger picture. Rajasi's guidance generally applies but I'm resolving to reduce noise.